lib/ex_aws/sqs/parsers.ex

```diff
@@ -0,0 +1,204 @@
+if Code.ensure_loaded?(SweetXml) do
+  defmodule ExAws.SQS.Parsers do
+    import SweetXml, only: [sigil_x: 2]
+
+    def parse({:ok, resp=%{body: xml}}, :list_queues) do
```

Comment:
I'm gonna be "that guy" and note that as a stylistic thing I tend to prefer `%{body: xml} = resp`